src/main/app/claims/responseModelConverter.ts

```diff
@@ -125,23 +127,25 @@ export class ResponseModelConverter {
         comment: draft.evidence.comment
       } as DefendantEvidence,
       freeMediation: FreeMediationUtil.convertFreeMediation(draft.freeMediation),
+      mediationPhoneNumber: mediationDraft.canWeUse.mediationPhoneNumber,
       defendant: this.convertPartyDetails(draft.defendantDetails),
       statementOfTruth: this.convertStatementOfTruth(draft)
     }
   }
 
-  private static convertFullAdmission (draft: ResponseDraft): FullAdmissionResponse {
+  private static convertFullAdmission (draft: ResponseDraft, mediationDraft: MediationDraft): FullAdmissionResponse {
     return {
       responseType: ResponseType.FULL_ADMISSION,
       freeMediation: FreeMediationUtil.convertFreeMediation(draft.freeMediation),
```

Comment:
Please wrap in feature toggle if statement. if mediation is not turned on it should use the response draft otherwise it should use the mediationDraft. Same for the other conversions